### ISSUE TITLE
state.log_history为空

### ISSUE THREAD

**Issue by njhouse365** [hiyouga/ChatGLM-Efficient-Tuning#228]
class WandbCallback(TrainerCallback):
    def __init__(self):
        super().__init__()

    def on_train_end(self, args: TrainingArguments, state: TrainerState, control: TrainerControl, **kwargs):
        # 在训练结束时进行初始化
        wandb.log({'epoch': 0, 'loss': 0, 'accuracy': 0})

    def on_epoch_begin(self, args: TrainingArguments, state: TrainerState, control: TrainerControl, **kwargs):
        # 在每个epoch开始时记录指标
        if "loss" not in state.log_history[-1]:
            return
        wandb.log(
            {'epoch': state.epoch + 1, 'loss': state.log_history[-1].get("loss", None),
             'learning_rate': args.learning_rate})

# start a new wandb run to track this script
    wandb.init(
        # set the wandb project where this run will be logged
        project="ChatGLM-Efficient-Tuning2",
        # track hyperparameters and run metadata
        config={
            "learning_rate": training_args.learning_rate,
            "batch_size": training_args.per_device_train_batch_size,
            "epochs": training_args.num_train_epochs,
        }
    )

trainer = Seq2SeqTrainerForChatGLM(
        finetuning_args=finetuning_args,
        model=model,
        args=training_args,
        tokenizer=tokenizer,
        data_collator=data_collator,
        callbacks=[WandbCallback()] if training_args.do_train else None,
        compute_metrics=ComputeMetrics(tokenizer) if training_args.predict_with_generate else None,
        **trainer_kwargs
    )

在87步时报错，原因是这个state.log_history为空；今天更新代码之后才有了这个错误，
大佬，求助
